cmd/notification.go

```diff
@@ -1309,3 +1309,83 @@ func RemoveListener(objAPI ObjectLayer, bucketName string, targetID event.Target
 
 	return saveConfig(ctx, objAPI, configFile, data)
 }
+
+// getLambdaInfo - return the status of notification targets
+func (sys *NotificationSys) getLambdaInfo(config *serverConfig) madmin.Target {
+	var amqps []madmin.AMQP
+	var ess []madmin.Elasticsearch
+	var Kafkas []madmin.Kafka
+	var mqtts []madmin.MQTT
+	var myss []madmin.MySQL
+	var nats []madmin.NATS
+	var nsqs []madmin.NSQ
+	var psqls []madmin.PostgreSQL
+	var redis []madmin.Redis
+	var webhooks []madmin.Webhook
+
+	for _, target := range sys.targetList.List() {
```

Comment:
Suggestion - We could marshal the target args by extending the target interface with `MarshalJSON()`. So that we can reduce some code

And the TargetInfo struct can have 
```
type TargetInfo struct {
           isActive   bool  `json:"isActive"`
           config string `json:"config"`
}
```

And this getLambdaInfo function can return `[]TargetInfo`

Reply:
And IMO we could simply print each target config as a string in mc side (`printMsg()`)

Reply:
done.